pkg/operator/apis/monitoring/v1/types.go

```diff
@@ -144,6 +144,9 @@ const CompressionGzip CompressionType = "gzip"
 type KubeletScraping struct {
 	// The interval at which the metric endpoints are scraped.
 	Interval string `json:"interval"`
+	// TLSInsecureSkipVerify disables verifying the target cert.
+	// This can be useful for clusters provisioned with kubeadm.
+	TLSInsecureSkipVerify bool `json:"tlsInsecureSkipVerify,omitempty"`
```

Comment:
is this part of refactor?

Reply:
Yea - I added that feature to kubelet scraping here. It's non-breaking and probably pretty fringe, but it helps us test in e2e for kubelet scraping.

I'll mark it out as a separate commit.